spaceship/lib/spaceship/tunes/iap_list.rb

```diff
@@ -48,24 +46,20 @@ class IAPList < TunesBase
         'appMaximumNumberOfCodes' => :app_maximum_number_of_codes,
         'isEditable' => :is_editable,
         'isRequired' => :is_required,
-        'canDeleteAddOn' => :can_delete_addon,
-        'iTunesConnectStatus' => :status_raw
+        'canDeleteAddOn' => :can_delete_addon
       })
 
       class << self
         def factory(attrs)
+          @status = Tunes::IAPStatus.get_from_string(attrs["iTunesConnectStatus"])
+
+          # Parse the type
+          @type = Tunes::IAPType.get_from_string(attrs["addOnType"])
```

Comment:
on the other classes i used this form:

```
attrs["iTunesConnectStatus"] = Tunes::IAPStatus.get_from_string(attrs["iTunesConnectStatus"])
          attrs["addOnType"] = Tunes::IAPType.get_from_string(attrs["addOnType"])
```

seems to work, is that ok?

i somehow missed that, because i had no spec in the iaplist for this 2 fields (added some)